dev-haskell/snap-loader-dynamic/snap-loader-dynamic-1.0.0.2-r1.ebuild

```diff
@@ -15,11 +15,14 @@ SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
-IUSE=""
+
+CABAL_CHDEPS=('mtl               >  2.0     && < 2.3' 'mtl > 2.0'
+			'time              >= 1.1     && < 1.10' 'time >= 1.1'
+			'unix            >= 2.2.0.0 && < 2.8' 'unix >= 2.2.0.0')
 
 RDEPEND=">=dev-haskell/directory-tree-0.10:=[profile?] <dev-haskell/directory-tree-0.13:=[profile?]
 	>=dev-haskell/hint-0.3.3.1:=[profile?] <dev-haskell/hint-0.10:=[profile?]
-	>dev-haskell/mtl-2.0:=[profile?] <dev-haskell/mtl-2.3:=[profile?]
+	>dev-haskell/mtl-2.0:=[profile?]
```

Comment:
Drop `mtl` here (you can leave it in `CABAL_CHDEPS`) (see https://github.com/gentoo-haskell/gentoo-haskell/issues/1386)